bin/ch/WScriptJsrt.cpp

```diff
@@ -490,9 +495,39 @@ bool WScriptJsrt::PrintException(LPCWSTR fileName, JsErrorCode jsErrorCode)
             LPCWSTR errorMessage = nullptr;
             size_t errorMessageLength = 0;
 
-            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"message", &messagePropertyId), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsGetProperty(exception, messagePropertyId, &messageProperty), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsStringToPointer(messageProperty, &errorMessage, &errorMessageLength), false);
+            LPCWSTR errorName = nullptr;
+            size_t errorNameLength = 0;
+
+            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"name", &namePropertyId), false);
```

Comment:
>    It is being done inside JavascriptError:EntryToString that is called from ChakraRTInterface::JsStringToPointer(messageProperty, &errorMessage, &errorMessageLength).

AFAIK `exception` needs to be converted to `string` anyways.( `JsConvertValueToString` )  
The question is, whether the output depends on `toString` call from `exception` object or `name: message` that we craft manually. For the sake of output consistency, I would go for the second. However if your suggestion is the general practice here, that is also fine.